### ISSUE TITLE
Dictionaries not converted properly

### ISSUE THREAD

**Issue by adamchainz** [asottile/setup-py-upgrade#12]
Hi @asottile 

Great package, using it to upgrade all my projects to declarative metadata right now.

I've had one issue with dictionaries beingn converted to their `repr()`:

```
$ cat setup.py
from setuptools import setup

setup(project_urls={'homepage': 'https://adamj.eu'})
$ setup-py-upgrade .
./setup.py and ./setup.cfg written!
$ cat setup.py
from setuptools import setup
setup()
$ cat setup.cfg
[metadata]
project_urls = {'homepage': 'https://adamj.eu'}
```

The expected output for dictionaries is `key = value`, one per line, like:

```
[metadata]
project_urls =
    homepage = https://adamj.eu
```

It's not a bother for me, I can fix my projects by hand. But it does lead to a slightly confusing error which users might need a little time to debug from the message and the setup.cfg docs.

Thanks,

Adam

**Comment by asottile** [comment]
this should be a pretty easy fix, I probably didn't have any examples when building it. 

there's some other things that get formatted in that way, probably just an oversight for project urls

setup-cfg-fmt probably has the same issue if I had to guess

**Comment by adamchainz** [comment]
PR is up.

`setup-cfg-fmt` doesn't appear to have the problem,  I can apply it to my test `setup.cfg` with a dict just fine. I think this is because the config parser loads it as a multiline string.

**Comment by asottile** [comment]
this has been released as part of 1.1.0 -- thanks again for the issue and patch!
